batch/batch/worker.py

```diff
@@ -81,30 +84,94 @@ def free(self, port):
         self.ports.put_nowait(port)
 
 
-async def docker_call_retry(f, *args, **kwargs):
-    delay = 0.1
+def docker_call_retry(timeout, name):
+    async def _wrapper(f, *args, **kwargs):
```

Comment:
I'm a bit ambivalent on this, but it seems overall simpler to have one argument list:
```
def docker_call_retry(timeout, name, f, *args, **kwargs):
    ...
```
Thoughts?

Reply:
Removes the need for `local_timeout`

Reply:
I think originally I wanted a default value for timeout which makes handling kwargs challenging. I'll make it simpler.